eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php

```diff
@@ -25,12 +25,13 @@ public static function dumpAssets(CommandEvent $event)
         $options = self::getOptions($event);
         $appDir = $options['symfony-app-dir'];
         $webDir = $options['symfony-web-dir'];
-        $env = isset($options['ezpublish-asset-dump-env']) ? $options['ezpublish-asset-dump-env'] : '';
+        // @deprecated: Use of ezpublish-asset-dump-env is deprecated, use standard SYMFONY_ENV instead.
```

Comment:
It would work in production since we use `SYMFONY_ENV`, but in dev we don't use `SYMFONY_ENV` but I would still like not to be interrupted and just use the default `dev` value.